### ISSUE TITLE
Node.js CPC membership for 2022-2023

### ISSUE THREAD

**Issue by Trott** [nodejs/TSC#1258]
CPC terms end this month (July 31). Node.js has two voting reps: Joe Sepi and me.

Last year, one rep was appointed by CommComm and one by TSC. CommComm is no longer a thing, so I propose that (at least for this year) TSC simply appoint both members for the next year.

I know that Joe Sepi wishes to be re-appointed. While it might be ideal to do a call for nominations, etc., given the short time frame and no process in place to do that, I propose that we dispense with that and simply re-appoint Joe.

I am happy to continue as the TSC representative on the CPC. However, if another member of the TSC wishes to step in and do it instead, I would be happy with that as well (as long as it is someone with a good attendance record for TSC meetings--the key thing is to be a link between CPC and TSC). I propose that we give TSC folks until the next TSC meeting to self-nominate, and then have a vote (if necessary).

## TL;DR

1. Let's re-appoint Joe Sepi for another year term on CPC. If any TSC members have objections, please raise them here (or privately on the TSC mailing list) before the next TSC meeting.
2. TSC members who would want to be a voting CPC member representing the TSC/Node.js, please self-nominate so we can appoint someone at the next TSC meeting.

@nodejs/tsc @joesepi

**Comment by mhdawson** [comment]
I'm +1 to the suggestion.

I can't find anywhere that we have written down a process for selecting CPC reps. I think the TSC making the appointments is reasonable and in line with what is documented in the CPC governance.

People can already join the CPC so I think people who are both from the Node.js project and have already shown an interest to be involved in the CPC should be top of our list of choices. Based on that re-appointing Joe makes sense.

**Comment by gireeshpunathil** [comment]
+1 to both the proposals.

is there a difference between the role definitions of the 2 reps? or they are identical (in terms of voting power, responsibilities, etc.)

**Comment by mcollina** [comment]
+1 to both proposals.

**Comment by Trott** [comment]
> is there a difference between the role definitions of the 2 reps? or they are identical (in terms of voting power, responsibilities, etc.)

They are identical.

**Comment by Trott** [comment]
Re-appointed @joesepi and @Trott in the TSC meeting today.
